general/auth/2fa.rst

```diff
@@ -0,0 +1,127 @@
+=========================
+Two-factor Authentication
+=========================
+
+Two-factor authentication ("2FA") is a good way to improve the
+security of an account, to make it less likely that an other person
+will manage to log in instead of you.
+
+Practically, it means storing a secret inside an *authenticator*
+(usually your cell phone) and exchanging a code from the authenticator
+when you try to log in.
+
+This means an attacker needs *both* to have guessed (or found) your
+password and to access (or steal) your authenticator, a more difficult
+proposition than either one or the other.
+
+Requirements
+============
+
+.. note:: These lists are just examples, they are not endorsements of
+          any specific software.
+
+If you don't already have one, you will need to choose an
+authenticator.
+
+Phone-based authenticators are the easiest and most common so we will
+assume you'll pick and install one on your phone, examples include
+`authy <https://authy.com/>`_, `freeotp
+<https://freeotp.github.io/>`_, `google authenticator
+<https://support.google.com/accounts/answer/1066447?hl=en>`_,
+`lastpass authenticator <https://lastpass.com/auth/>`_, `microsoft
+authenticator
+<https://www.microsoft.com/en-gb/account/authenticator?cmp=h66ftb_42hbak>`_,
+...; password managers also commonly include :abbr:`2FA (two-factor
+authentication)` support e.g. `1password
+<https://support.1password.com/one-time-passwords/>`_, `bitwarden
+<https://bitwarden.com/help/article/authenticator-keys/>`_, ...
+
+For the sake of demonstration we will be using Google Authenticator
+(not because it is any good but because it is quite common).
```

Comment:
Screens should look quite similar in other apps

Reply:
Yes but details might vary quite a lot.

Reply:
I have changed screenshots to enterprise version but did not changed the screenshots of GA because this shit ain't touching my phone and authy doesn't allow screenshots....